hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -94,14 +97,26 @@ module clkmgr import clkmgr_pkg::*; (
 % endfor
 
 % for src in div_srcs:
+
+  lc_ctrl_pkg::lc_tx_t ${src['name']}_div_scanmode;
```

Comment:
this looks good, just a note here that we may have to make these signals degenerate packed arrays `[0:0]`.
I believe @cindychip has experienced some FPV tooling issues with the `prim_lc_sync` module when the `lc_en_o` output is not connected to a packed array...